docs/reference/actions.md

```diff
@@ -0,0 +1,12 @@
+# Actions
+
+- _get-initial-password_: Retrieve auto-generated initial password for accessing WordPress admin
```

Comment:
Action name in the actions.yaml is `get-initial-password`

Reply:
May I know if you mean the markdown syntax for italics _action-name_ is confusing to read? I could update that with *action-name* or get rid of the itatlics formatting altogether if it reads better :)
Thank you for your comment!

Reply:
Nvm me, I just noticed those are italics